Snowflake.Data/Core/Session/SFSessionHttpClientProperties.cs

```diff
@@ -75,7 +75,7 @@ internal TimeSpan TimeoutDuration()
         internal HttpClientConfig BuildHttpClientConfig()
         {
             return new HttpClientConfig(
-                insecureMode,
+                !insecureMode,
```

Comment:
Why would we even want to allow insecureMode?

Reply:
we allow it in _every_ Snowflake driver. the default setting is (except for this bug) to use certificate validation (OCSP / CRL) but we need to give the user the option , that knowing the risk and accepting it on their full own responsibility, to turn it off.
We never recommend to do it though, other than very well outlined, short test purposes.

Reply:
more on this here https://community.snowflake.com/s/article/How-to-turn-off-OCSP-checking-in-Snowflake-client-drivers